cypress/support/commands.js

```diff
@@ -76,6 +76,7 @@ Cypress.Commands.add('fillReviewInformation', user => {
   const { email, line, rating, name, review } = user
 
   cy.get(rrselectors.formBottomLine).clear().type(line)
+  cy.get(rrselectors.emptyStars).should('not.have.length', 5)
```

Comment:
Are you sure ?
if length gets changed this will get fail

Reply:
yes because default stars are allowed only between 1 and 5 and there is no chance it is set to zero. So we will not get 5 empty star.